### DISCUSSION TITLE
Regional "Badge"

### DISCUSSION THREAD

**Discussion by wangzhenux** [twilio-labs/paste#2369] [Office hours] [answered]
**Background:** 
We used to talk to the DSYS team to develop a badge for Regional to indicate the configuration element in console is regionalized but back then, we don't have badge components, so we customized it with popoverbutton. 

**The specific questions I have is**
As today, we have badge component, so we'd like to adopt it while we need to understand the best practice when there are other badges available on the page, and what color should be used.

![image](https://user-images.githubusercontent.com/6403036/165122503-eb9cd6db-a058-4d5a-83d4-84893c1a7be2.png)
![image](https://user-images.githubusercontent.com/6403036/165122688-93ce6922-8ecf-4338-b5ff-eed0c142ed8a.png)

**Comment by shleewhite** [comment, marked as answer]
Thanks for coming to office hours @wangzhenux! 

To summarize what we talked about: the blue looks good. If you do user research on this flow, please let us know the results!
